src/components/radios/_radios.scss

```diff
@@ -99,7 +100,12 @@
 
   // Focused state
   .govuk-radios__input:focus + .govuk-radios__label::before {
-    box-shadow: 0 0 0 4px $govuk-focus-colour;
+    top: -($govuk-radios-focus-width);
```

Comment:
I think this makes sense, but could definitely do with a comment to explain what's going on here.